### ISSUE TITLE
Replicate ID's - allow more than a single character

### ISSUE THREAD

**Issue by dsurujon** [jsa-aerial/aerobio#13]
Looks like the replicate IDs can only be a single character, which limits the number of possible replicates in an experiment. For large (e.g. human population) studies there might be hundreds of samples in one group, so it would be nice to have replicate ID's that can be more than a single character. A workaround for now is to do things in batches of 26 and name replicates a-z.

**Comment by jsa-aerial** [comment]
Yes, I talked about this with Jackob over the weekend.  One possibility would be to support 'arbitrary' counts if numbers are used for replicate id.  It is already recommended to not mix letters and digits but stick to one or the other.  This could be enforced and then add support for number repids that can be of any cardinality.  A further constraint would likely be needed that these would need to be in sequence without gaps.  Would that work?

**Comment by dsurujon** [comment]
Yeah I think that would be great. Since replicates within one group would (probably) be named arbitrarily, it should be no problem to assign numbers in sequence with no gaps
